test/unit_tests/apps/youth-employment-success/js/budget-form-view-spec.js

```diff
@@ -110,7 +110,7 @@ describe( 'BudgetFormView', () => {
     expect( totalEl.textContent ).toEqual( '0.00' );
   } );
 
-  it( 'defaults the `total` value to zero when no input has been received', () => {
+  it.only( 'defaults the `total` value to - when no input has been received', () => {
```

Comment:
Should this have `.only` or was this just for testing?

Reply:
Nope! Just for testing, great catch.